srcpkgs/vscode/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'vscode'
 pkgname=vscode
-version=1.63.2
+version=1.65.2
 revision=1
-_electronver=13.6.1
+_electronver=13.6.7
 hostmakedepends="pkg-config python nodejs yarn tar git ripgrep"
-makedepends="libxkbfile-devel libsecret-devel electron13"
-depends="libXtst libxkbfile nss dejavu-fonts-ttf xdg-utils ripgrep electron13"
+makedepends="libxkbfile-devel libsecret-devel libxml2-devel ncurses-devel electron13"
+depends="libXtst libxkbfile ncurses nss dejavu-fonts-ttf xdg-utils ripgrep electron13"
```

Comment:
```suggestion
depends="libXtst ncurses nss dejavu-fonts-ttf xdg-utils ripgrep electron13"
```
It should be auto-detected as a shlib dependency.

Reply:
Works for me.